requirements/requirements.txt

```diff
@@ -1,7 +1,12 @@
+hatchling 
 huggingface_hub>=0.24
+llama-index==0.10.43
+nemo_run @ git+https://github.com/NVIDIA/NeMo-Run.git@${NEMO_RUN_TAG:-main}
 numba
 numpy>=1.22
+nvidia-modelopt[torch]==${MODELOPT_VERSION:-0.19.0}
```

Comment:
Remove from core requirements. You may put it for export requirements.

Reply:
is vllm a good place?